packages/main/src/Form.ts

```diff
@@ -59,6 +59,9 @@ type ItemsInfo = {
  * The Form is a layout component that arranges labels and form fields (like input fields) pairs
  * into a specific number of columns.
  *
+ * **Note:** The Form web component is a layout, it isn't a replacement for the `form` HTML element.
```

Comment:
The Form web component is a layout component, it isn't a replacement for the native `form` HTML element.